data/data/openstack/variables-openstack.tf

```diff
@@ -1,3 +1,15 @@
+variable "openstack_master_root_volume_type" {
+  type        = string
+  default     = null
+  description = "The type of volume for the root block device of master nodes."
+}
+
+variable "openstack_master_root_volume_size" {
+  type        = number
+  default     = null
```

Comment:
i don't think null is required as default here. see https://www.terraform.io/docs/configuration/expressions.html#null